CommonControls/Assimp/AssimpImporter.cs

```diff
@@ -0,0 +1,350 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Text;
+using CommonControls.FileTypes.RigidModel.LodHeader;
+using CommonControls.FileTypes.RigidModel.MaterialHeaders;
+using CommonControls.FileTypes.RigidModel;
+using CommonControls.FileTypes.RigidModel.Vertex;
+using Assimp;
+using CommonControls.FileTypes.Animation;
+using System.Windows.Forms;
+using Filetypes.ByteParsing;
+using CommonControls.FileTypes.PackFiles.Models;
+using CommonControls.Services;
+using Serilog;
+using CommonControls.Common;
+
+namespace CommonControls.ModelImportExport
+{    public class AssimpImporter
+    {
+        ILogger _logger = Serilog.Log.ForContext<AssimpImporter>();
+
+        private Assimp.Scene _assScene;
+        private AnimationFile _skeletonFile;
+        private PackFileService _packFileService;
+
+        public AssimpImporter(PackFileService packFileService)
+        {
+            _packFileService = packFileService;
+        }
+        public void ImportScene(string fileName)
+        {
+            ImportAssimpScene(fileName);
+        }
+        private string GetSkeletonString()
+        {
+            string tempSkeletonString = "";
+
+            var parent = _assScene.RootNode;
+
+            SearchNodesRecursiveLocal(parent, ref tempSkeletonString);
+
+            return tempSkeletonString;
+
+            void SearchNodesRecursiveLocal(Node parent, ref string skeletonString)
+            {
+                foreach (var node in parent.Children)
+                {
+                    if (node.Name.Contains("skeleton"))
+                    {
+                        skeletonString = node.Name.Replace("skeleton//", "");
+                    }
+
+                    if (skeletonString.Length > 0)
+                    {
+                        return;
+                    }
+
+                    SearchNodesRecursiveLocal(node, ref skeletonString);
+                }
+            }
+        }
```

Comment:
Missing line break. Always make sure its one line between each function to make it clear where something starts and stops